.circleci/workflows.yml

```diff
@@ -177,7 +177,7 @@ jobs:
     executor: vm
     steps:
       - initialize-environment
-      - run: poetry run ape test beamer/tests/{agent,health} -s --gas --cov beamer --cov-report=term
+      - run: poetry run ape test beamer/tests/{agent,health} -s --gas -k "not slave" --cov beamer --cov-report=term
```

Comment:
This is not necessary, assuming you rename the test, as I suggested in my comment.